cmd/gitService.go

```diff
@@ -0,0 +1,119 @@
+package cmd
+
+import (
+	_ "embed"
+	"os"
+	"os/exec"
+
+	log "github.com/sirupsen/logrus"
+)
+
+var DEPLOY_CONFIG_GIT_REPO string = "https://automotivemastermind@dev.azure.com/automotivemastermind/aM/_git/am.devops.deploy"
+var DEPLOY_CONFIG_GIT_REPO_BRANCH string = "local"
+
+var HELM_CONFIG_GIT_REPO string = "https://automotivemastermind@dev.azure.com/automotivemastermind/aM/_git/am.devops.helm"
+var HELM_CONFIG_GIT_REPO_BRANCH string = "local"
+
+//var FPS string = string(filepath.Separator)
+
+func setUpLocalGitFolder(folderName string, clusterRootPath string, clusterName string) {
+
+	moveCloneIntoLocalRepo(folderName, clusterRootPath)
+
+	commandExec := exec.Command("git", "init")
+
+	commandExec.Dir = clusterRootPath + FPS + folderName
```

Comment:
looks like these are joined multiple times in this function. might as well do it once and store it in a variable and use that variable moving forward.